### PR TITLE
Feature/Create Team Section

### PR DESCRIPTION
**Pull request by annapodoliaka**
Hi :)
I created a new page "Team". Adapted the page for 2 main breakpoints-mobile and desktop, for the other of the breakpoints I changed the components as necessary.
Also added adaptive images and compressed the logo.

### CODE REVIEW

**Comment by annapodoliaka** [comment]
Forgot to add "loading lazy" and image description (alt), and also dont add width and height to image. 
And I reviewed the page again - it's not very comfortable to read the code in the  tag <picture>
:(

**Comment by Plakhotenko** [comment]
> Forgot to add "loading lazy" and image description (alt), and also dont add width and height to image.
> And I reviewed the page again - it's not very comfortable to read the code in the tag
> :(

Don't worry. You can improve it in the current PR.

File: styles/_contacts.scss [hunk 1]
```diff
@@ -179,7 +179,9 @@
   @include media-breakpoint-up(xl) {
     font-size: 32px;
     line-height: (32 / 38);
-    color: linear-gradient(#F0145A 100%, #EF8080 100%);
+    background-image: linear-gradient(to right,#F0145A, #EF8080);
+    color: transparent;
+    -webkit-background-clip: text;
```
Comment (Plakhotenko) [inline styles/_contacts.scss:184]:
```suggestion
    -webkit-background-clip: text;
    background-clip: text;
```

Comment (Plakhotenko) [reply styles/_contacts.scss:184]: Do not forget to add property without prefix.

File: styles/_team.scss [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  }
+}
+.title-team-section {
+  display: block;
+  font-size: 28px;
+  line-height: (38 / 32);
+  font-weight: 600;
+  background-image: linear-gradient(to right,#F0145A, #EF8080);
+  color: transparent;
+  -webkit-background-clip: text;
```
Comment (Plakhotenko) [inline styles/_team.scss:22]: Also add property without prefix.

File: styles/_team.scss [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    padding: 82px 106px 80px 150px;
+  }
+}
+.nav-team-section {
+  @include media-breakpoint-up(xl) {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+.nav-team-section__list-item-1 {
```
Comment (Plakhotenko) [inline styles/_team.scss:46]: maybe just `.nav-team-section__list-item`?

File: styles/_team.scss [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  align-items: end;
+  @include media-breakpoint-up(xl) {
+    flex-direction: row;
+    align-items: baseline;
+  }
+}
+.team-member {
+  font-size: 16px;
+  line-height: (21 / 18);
+  font-weight: 500;font-size: 16px;
```
Comment (Plakhotenko) [inline styles/_team.scss:130]:
```suggestion
  font-weight: 500;
  font-size: 16px;
```

File: styles/_team.scss [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  color: $main-color;
+}
+.team-member1 {
```
Comment (Plakhotenko) [inline styles/_team.scss:133]: Why don't just `team-member`?

File: team.html [hunk 6]
```diff
@@ -0,0 +1,428 @@
...
+                        srcset="./images/maiboroda-desk-1x.webp 1x, 
+                                ./images/maiboroda-desk-2x.webp 2x" type="image/webp">
+                <source media="(min-width: 1400px)" 
+                        srcset="./images/maiboroda-desk-1x.jpg 1x, 
+                                ./images/maiboroda-desk-2x.jpg 2x" type="image/jpeg">
+                <source srcset="./images/maiboroda-mob-1x.webp 1x, 
+                                ./images/maiboroda-mob-2x.webp 2x" type="image/webp">
+                <source srcset="./images/maiboroda-mob-1x.jpg 1x, 
+                                ./images/maiboroda-mob-2x.jpg 2x" type="image/jpeg">
+                <img class="img-team-section" src="./images/maiboroda-desk-1x.jpg" alt="">
```
Comment (Plakhotenko) [inline team.html:133]: Add value for alt.

File: team.html [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <section class="footer-section">
+      <div class="container d-flex flex-direction-column align-items-center container-desktop-footer">
+        <div class="group-address-contacts-section-contacts">
+          <div class="group-address-contacts">
+            <h1 class="title-footer">
```
Comment (Plakhotenko) [inline team.html:357]: Are you sure about h1 at this point?

File: styles/_contacts.scss (hunk 1 above)
Comment (annapodoliaka) [reply styles/_contacts.scss:184]: Without prefix this property dont work

File: styles/_team.scss (hunk 3 above)
Comment (annapodoliaka) [reply styles/_team.scss:46]: .nav-team-section__list-item and .nav-team-section__list-item-1 contains different elements, and each of elements need different properties

File: styles/_team.scss (hunk 4 above)
Comment (annapodoliaka) [reply styles/_team.scss:130]: Fixed

File: styles/_team.scss (hunk 5 above)
Comment (annapodoliaka) [reply styles/_team.scss:133]: It was necessary to make a smaller font-size for long last name of one of the taem members.

File: team.html (hunk 6 above)
Comment (annapodoliaka) [reply team.html:133]: Added for all images

File: team.html (hunk 7 above)
Comment (annapodoliaka) [reply team.html:357]: Fixed it on tag span

**Comment by annapodoliaka** [comment]
I made changes in the current branch on which I am located (create_about_section)
Can I merge branch create_team_section?

**Pull request merged**